package/MDAnalysis/topology/tpr/obj.py

```diff
@@ -112,6 +114,21 @@ def __init__(self, id, name, type, resid, resname, mass, charge):
         self.resname = resname
         self.mass = mass
         self.charge = charge
+        self.atomic_number = atomic_number
+
+    @property
+    def element_symbol(self):
+        """
+        The symbol of the atom element.
+
+        Returns
```

Comment:
It's odd to think of a property "returning" something, but on the other hand very few people will see this